### ISSUE TITLE
Aerial starts when i'm on Youtube fullscreen mode

### ISSUE THREAD

**Issue by larrix** [OrangeJedi/Aerial#161]
Bug: 

I got the settings on "Start Aerial After 5 minutes" and I had no problems with anything else, but when i'm on Youtube (on Chrome) in fullscreen mode, Aerial starts runnnig and I have to move the mouse or whatever to continue watching the video. Is annoying as hell. 
Please would you look into this?

Aerial Version: 
1.2.1

**Comment by grendizerus** [comment]
Hello I have the same problem when I'm watching a video with Pot Player.
The [version history page](https://github.com/OrangeJedi/Aerial/wiki/Version-History) says I should run Aerial in administrator mode. I tried that and when I restart Windows, Aerial just won't load.

Should I install Aerial as a scr screensaver? If I do that, do I need to uninstall the stand alone Aerial first? Why would the scr version have performance issues?
Thanks

DS

**Comment by antoniomika** [comment]
Pretty sure the logic here is a bit skewed: https://github.com/OrangeJedi/Aerial/blob/master/app.js#L1118

1. execSync needs to be used if we're not using promises as the callback return won't make it out of that closure. 
2. `powercfg /requests` returns 6 instances of `None` if there *isn't* a video playing. The logic in the callback will return true when a video is playing because there are not 6 instances of None in the command output which then will still allow the screensaver to start.

@OrangeJedi are you accepting contributions on this project? I saw there was a PR open for some bugfixes and would be happy to commit my own fixes if you're accepting them.

**Comment by OrangeJedi** [comment]
Good catch! I would happily accept a PR. I haven't accepted any recently because I haven't had time to work on the project, but I hope too soon.

**Comment by CthulhuGits** [comment]
Any progress on a fix for this? Bug is driving me bonkers. Have to keep turning off the SS constantly while watching vids.

**Comment by grendizerus** [comment]
bah perhaps this project is abandoned? Too bad it was one of the bet screensaver.
